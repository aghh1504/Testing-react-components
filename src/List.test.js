import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import List from './List';


it('should render a todo item ', () => {
  const tree = toJson(shallow(<List items={[]} />));
  expect(tree).toMatchSnapshot();
});
