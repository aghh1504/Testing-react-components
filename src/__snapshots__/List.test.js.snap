exports[`test should render a todo item  1`] = `<ul />`;
